third_party/ruby_externals.bzl

```diff
@@ -10,8 +10,8 @@ def register_ruby_dependencies():
 
     http_file(
         name = "rubygems_update_stripe",
-        urls = _rubygems_urls("rubygems-update-3.1.2.gem"),
-        sha256 = "7bfe4e5e274191e56da8d127c79df10d9120feb8650e4bad29238f4b2773a661",
+        urls = _rubygems_urls("rubygems-update-3.2.33.gem"),
```

Comment:
Does the `rubygems-update` gem contain `bundler`?

Reply:
Yes.